build-utils/css/utils/__tests__/convertToCssVariable.test.ts

```diff
@@ -0,0 +1,295 @@
+import { theme } from '../../constants/theme';
+import { convertToCssConstant } from '../convertToCssVariable';
+
+type UnitTestItem = {
+  case: string;
+  input: Record<string, string>;
+};
+
+const testCases: UnitTestItem[] = [
+  {
+    case: `Spacing constants`,
+    input: theme,
+  },
+];
+
+test.each(testCases)(`Case $case:`, ({ input }) =>
+  expect(convertToCssConstant(input)).toMatchInlineSnapshot(`
```

Comment:
This inline snapshot wont change if the CSS changes right? so if just one property is added it will break the test?

Reply:
yes, correct. If you add 1 new prop or adjust an existing prop. It will break when you run the test, and you need to adjust it either manually if you run test without key `--watchAll`, or update the snapshot interactively with built-in jest when running `jest --watchAll`